api/app/controllers/spree/api/base_controller.rb

```diff
@@ -73,8 +73,7 @@ def error_during_processing(exception)
 
         error_notifier.call(exception, self) if error_notifier
 
-        render plain: { exception: exception.message }.to_json,
-          status: 422
+        render json: { exception: exception.message }, status: 422
```

Comment:
Should we add this (and the others in "Use render :json insteda of render :text for json") to the non-Rails 5 branch also?